types/nightmare/index.d.ts

```diff
@@ -96,6 +96,7 @@ declare class Nightmare {
     removeListener(event: 'error', cb: (msg: string, trace?: Nightmare.IStackTrace[]) => void): Nightmare;
     removeListener(event: 'timeout', cb: (msg: string) => void): Nightmare;
     screenshot(path: string): Nightmare;
+    screenshot(path: string, clip: Object): Nightmare;
```

Comment:
Thanks @DanielRosenwasser for the feedback.  I have adjusted and added overloads to handle the buffer being returned too.